config.md

```diff
@@ -167,8 +167,8 @@ _Note: symbolic name for uid and gid, such as uname and gname respectively, are
 
 ## Platform-specific configuration
 
-* **`os`** (string, required) specifies the operating system family this image must run on. Values for os must be in the list specified by the Go Language document for [`$GOOS`](https://golang.org/doc/install/source#environment).
-* **`arch`** (string, required) specifies the instruction set for which the binaries in the image have been compiled. Values for arch must be in the list specified by the Go Language document for [`$GOARCH`](https://golang.org/doc/install/source#environment).
+* **`os`** (string, required) specifies the operating system family this image MUST run on. Values for os MUST be in the list specified by the Go Language document for [`$GOOS`](https://golang.org/doc/install/source#environment).
```

Comment:
What sort of requirement is that first MUST?  Is it “the runtime MUST generate an error if invoked with a config.json whose **`os`** is incompatible with the host platform”?  Or is it “the bundle-validator MUST generate an error if **`os`** is incompatible with the content the bundle's other content (e.g. `linux` in **`os`**, but only Windows binaries in the bundle's rootfs)?  I think the first sentence needs to be reworded to make the condition less ambiguous, or we should reword it to just be informative (e.g. “… image runs on.”).

For the second MUST, I doubt we want to require a compliant runtime support all Go architectures itself.  And there is a benefit to pointing runtime/bundle authors at the Go set, but not much benefit in making that a hard limit (see also opencontainers/image-spec#29).  Maybe reword that sentence to something like:

> The runtime MUST generate an error if it does not support the configured **`os`**.
> Bundle authors SHOULD use—and runtime authors SHOULD understand—**`os`** entries listed in the Go Language document for [`$GOOS`](https://golang.org/doc/install/source#environment).